geoips/plugins/modules/colormaps/pmw_tb/cmap_37pct.py

```diff
@@ -15,10 +15,12 @@
 
 LOG = logging.getLogger(__name__)
 
-cmap_type = "linear_segmented"
+interface = "colormaps"
+family = "linear_segmented"
+name = "pmw_37pct"
```

Comment:
@jsolbrig Identical name to algorithms.pmw_37pct

Reply:
I think this is fine for now. We'll address when we address #204.